.github/workflows/ubuntu.yml

```diff
@@ -45,24 +63,18 @@ jobs:
           libxfixes-dev \
           libwayland-dev \
           meson
-    - name: print compiler version
+    - name: 'Print compiler version'
       run: ${{ matrix.compiler }} --version
-    - name: meson setup
-      if: ${{ (matrix.build == 'meson') }}
-      run: meson setup ./builddir --prefix=/usr
-    - name: meson compile
+    - name: 'Configure (meson)'
       if: ${{ (matrix.build == 'meson') }}
-      run: meson compile -C ./builddir || ninja -C ./builddir
-    - name: meson install
+      run: meson setup _build --prefix=/tmp/libva/usr
+    - name: 'Build and Install (meson)'
```

Comment:
I am not OK to collapse make and make install stages. I believe 2 stages approach better reflect what people are doing hence has more chances to catch issues.

Reply:
Sure thing - will split them with v2.

Out of curiosity: do you have any issues in mind, or this is more of a "just in case"?